docs/docs/layouts/modal.md

```diff
@@ -0,0 +1,17 @@
+A large size module with an overlay behind it - meant to cover a substantial area of the browser window, so that it demands attention from the user.
+
+## variant
+
+Variants determines any extra content that may be used by the module.
+
+| Value | Type | Behavior |
+|---|---|---|
+| 1 | int | `default` text-only module |
+| 2 | int | module includes an image |  
+| 3 | int | module includes a content recommendation |  
```

Comment:
I'm not a fan of this API, but that's not what I'm reviewing, so I'll zip my mouth.